lib/game.rb

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
+
+require_relative 'board'
+
+class Game
+  attr_reader :board, :active_player
+
+  def initialize
+    @board = Board.new
+    @active_player = :white
+    @players = %i[white black]
+  end
+
+  def compute_path(figure, start, stop)
```

Comment:
Please put the private methods behind a `private`. Currently it's fairly hard to understand what's going on in here.
Added benefit is that the custom is that you only test public methods.

The `compute_path` method would definitely use some comments explaining what you are doing and why